test/remote/gateways/remote_card_stream_test.rb

```diff
@@ -113,6 +113,17 @@ def setup
       month: '12',
       year: '2020',
       brand: :visa)
+
+    @visacredit_three_ds_options = {
+      threeds_required: true,
+      three_d_secure: {
+        enrolled: 'true',
+        authentication_response_status: 'Y',
+        eci: '05',
+        cavv: 'Y2FyZGluYWxjb21tZXJjZWF1dGg',
+        xid: '00000000000004717472'
```

Comment:
same here.

Reply:
For remote tests using a 3ds2 styled uuid would break remote the tests because It should be a real ds transaction identifier  value, otherwise the CardStream will respond with invalid threeDSXID

Reply:
😩  I see, thanks for the explanation.